nft_ingester/src/bin/ingester/main.rs

```diff
@@ -472,6 +476,31 @@ pub async fn main() -> Result<(), IngesterError> {
         });
     }
 
+    // Start the management API if port is provided
+    if let Some(port) = args.management_api_port {
+        info!("Starting Management API...");
+        let cloned_rocks_storage = primary_rocks_storage.clone();
+        let cloned_rpc_client = rpc_client.clone();
+        let metrics = metrics_state.ingester_metrics.clone();
+        let postgre_client = index_pg_storage.clone();
+        let well_known_fungible_accounts = well_known_fungible_accounts.clone();
+        usecase::executor::spawn({
+            let cancellation_token = cancellation_token.child_token();
+            async move {
+                nft_ingester::management_api::start_management_api(
+                    cancellation_token,
+                    metrics,
+                    cloned_rocks_storage,
+                    postgre_client,
+                    cloned_rpc_client,
+                    well_known_fungible_accounts,
+                    port,
+                )
+                .await
+            }
+        });
+    }
```

Comment:
nit: no need for renames if these clones are moved into the block inside of spawn (L488)